React/game/src/app/components/App/index.js

```diff
@@ -9,11 +9,20 @@ import Login from '../../screens/Login/components/Login/index';
 class App extends Component {
   render() {
     const isLogin = this.props.isLogin;
-    const validateRoute = () => (isLogin ? <Redirect to="/game" /> : <Login />);
+    const userId = this.props.userId;
+    const validateRoute = props => {
+      const pathName = props.location.pathname;
+      if (isLogin && userId === 1) {
+        return <Game />;
+      } else if (!isLogin && pathName != '/login') {
+        return <Login />;
+      }
+      return <Login />;
+    };
```

Comment:
By doing `return <Login />` and `return <Game />`, you're not actually using the routing for well... routing the different screens. Try to find a way to use the routing to create the desired behaviour

Reply:
👍